examples/skills_tuning/instructlab/table_manipulation.py

```
@@ -0,0 +1,639 @@
+
+# %%
+# %load_ext autoreload
+# %autoreload 2
+
+# %%
+# Standard
+from importlib import resources
+import random
+
+# Third Party
+from blocks import *
```

Flake8 (7.2.0)

[error] 'from blocks import *' used; unable to detect undefined names

(F403)